Modelica/Utilities/Streams.mo

```diff
@@ -120,7 +120,7 @@ separated by LF or CR-LF.
   pure function error "Print error message and cancel all actions - in case of an unrecoverable error"
     extends Modelica.Icons.Function;
     input String string "String to be printed to error message window";
-    external "C" ModelicaError(string) annotation(Include="#include \"ModelicaUtilities.h\"", Library="ModelicaExternalC");
```

Comment:
At the moment, it is a tool responsibility to ensure `#include "ModelicaUtilities.h"` will find the file, so doesn't this change belong in a separate discussion about how Modelica external functions should point to _ModelicaUtilities.h_?